doc/release-notes.md

```diff
@@ -70,6 +70,7 @@ RPC changes
   /rest/block/ endpoints when in json mode. This is also included in `getblock`
   (with verbosity=2), `listsinceblock`, `listtransactions`, and
   `getrawtransaction` RPC commands.
+- In `getmempoolancestors`, `getmempooldescendants`, `getmempoolentry` and `getrawmempool` RPCs `size` has been renamed to `vsize` to be consistent with the returned value and other RPCs such as `getrawtransaction`.
```

Comment:
Please update note, something like ``In ... `vsize` was added and `size` is now deprecated ...``